front/app/services/users.ts

```diff
@@ -139,3 +153,40 @@ export async function completeRegistration(
 
   return authUser;
 }
+
+export function blockedUsersCount(streamParams: IStreamParams | null = null) {
```

Comment:
Minor and optional: Might be good to use react-query if you have the time. This is out of scope for this one so I'll leave it to you